guava/src/com/google/common/base/Optional.java

```diff
@@ -121,7 +124,7 @@
    * @since 21.0
    */
   @NullableDecl
-  public static <T> Optional<T> fromJavaUtil(@NullableDecl java.util.Optional<T> javaUtilOptional) {
+  public static <T extends Object> @PolyNull Optional<T> fromJavaUtil(@NullableDecl java.util.Optional<T> javaUtilOptional) {
```

Comment:
It usually doesn't make sense to write just one occurrence of `@PolyNull`.
See https://checkerframework.org/manual/#qualifier-polymorphism-multiple-qualifiers and https://checkerframework.org/manual/#qualifier-polymorphism-single-qualifier .
I suspect you will want to change the annotations on this method and the next one.

Reply:
I'll make the changes as suggested